client/starter-app/src/app/views/pages/login/login.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import {Observable} from 'rxjs';
+import { Headers, RequestOptions } from '@angular/http';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/toPromise';
+import { Http, Response } from '@angular/http';
+
+
+@Injectable()
+export class LoginService {
+
+  constructor(private http:Http) { }
+
+  postLoginData(body) {
+    console.log("body",body)
+    return this.http.post('http://10.0.1.62:8080/api/customer/login',body)
```

Comment:
store the URL separately in a variable 
for example, 
private serverURL = 'http://10.0.1.62:8080/api/'

also please remove console
handle errors when returning the post. look out for the response status